docs/content/how-to/kubevirt/create-kubevirt-cluster.md

```diff
@@ -53,31 +86,59 @@ export PULL_SECRET="$HOME/pull-secret"
 export MEM="6Gi"
 export CPU="2"
 export WORKER_COUNT="2"
+export BASE_DOMAIN=hypershift.lab
```

Comment:
we don't want to specify a basedomain by default. that's the advanced case that requires someone to setup DNS entries

Reply:
Changed this.